[Bug] Cannot npm run dev

**To Reproduce**
Hello! I've created `.env` file with cliend id and secret then cloned the repo with `git clone git@github.com:taniarascia/takenote` then `npm i`, transfered `.env` file to the root of the project, after that `npm run dev`.

**Expected behavior**
Working locally project.

**Actual behavior**
Getting an error:
```
Asd@DESKTOP-L18UDLE MINGW64 /c/Test/takenote (master)
$ npm run dev

> takenote@0.7.2 dev C:\Test\takenote
> concurrently 'npm run server' 'npm run client'

[0] "'npm" я┐╜я┐╜ явля┐╜я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜реня┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜шнея┐╜
[0] я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜, я┐╜споя┐╜я┐╜яемя┐╜я┐╜ я┐╜рограмя┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜ файя┐╜я┐╜я┐╜.
[1] "run" я┐╜я┐╜ явля┐╜я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜реня┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜шнея┐╜
[1] я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜, я┐╜споя┐╜я┐╜яемя┐╜я┐╜ я┐╜рограмя┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜ файя┐╜я┐╜я┐╜.
[2] "server'" я┐╜я┐╜ явля┐╜я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜реня┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜шнея┐╜
[2] я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜, я┐╜споя┐╜я┐╜яемя┐╜я┐╜ я┐╜рограмя┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜ файя┐╜я┐╜я┐╜.
[3] "'npm" я┐╜я┐╜ явля┐╜я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜реня┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜шнея┐╜
[3] я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜, я┐╜споя┐╜я┐╜яемя┐╜я┐╜ я┐╜рограмя┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜ файя┐╜я┐╜я┐╜.
[4] "run" я┐╜я┐╜ явля┐╜я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜реня┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜шнея┐╜
[4] я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜, я┐╜споя┐╜я┐╜яемя┐╜я┐╜ я┐╜рограмя┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜ файя┐╜я┐╜я┐╜.
[5] "client'" я┐╜я┐╜ явля┐╜я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜реня┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜шнея┐╜
[5] я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜, я┐╜споя┐╜я┐╜яемя┐╜я┐╜ я┐╜рограмя┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜ я┐╜я┐╜я┐╜я┐╜я┐╜я┐╜ файя┐╜я┐╜я┐╜.
[3] 'npm exited with code 1
[2] server' exited with code 1
[1] run exited with code 1
[0] 'npm exited with code 1
[5] client' exited with code 1
[4] run exited with code 1
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! takenote@0.7.2 dev: `concurrently 'npm run server' 'npm run client'`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the takenote@0.7.2 dev script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\Asd\AppData\Roaming\npm-cache\_logs\2020-11-13T14_47_22_267Z-debug.log
```

debug.log is:
```0 info it worked if it ends with ok
1 verbose cli [
1 verbose cli   'C:\\program files\\nodejs\\node.exe',
1 verbose cli   'C:\\Users\\Asd\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js',
1 verbose cli   'run',
1 verbose cli   'dev'
1 verbose cli ]
2 info using npm@6.14.8
3 info using node@v12.18.3
4 verbose run-script [ 'predev', 'dev', 'postdev' ]
5 info lifecycle takenote@0.7.2~predev: takenote@0.7.2
6 info lifecycle takenote@0.7.2~dev: takenote@0.7.2
7 verbose lifecycle takenote@0.7.2~dev: unsafe-perm in lifecycle true
8 verbose lifecycle takenote@0.7.2~dev: PATH: C:\Users\Asd\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\node-gyp-bin;C:\Test\takenote\node_modules\.bin;C:\Users\Asd\bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\local\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\usr\bin;C:\Program Files\Git\mingw64\bin;C:\Program Files\Git\usr\bin;C:\Users\Asd\bin;C:\program files (x86)\common files\oracle\java\javapath;C:\windows\system32;C:\windows;C:\windows\system32\wbem;C:\windows\system32\windowspowershell\v1.0;C:\windows\system32\openssh;C:\program files\git\cmd;C:\program files\nodejs;C:\program files\nvidia corporation\nvidia ngx;C:\WINDOWS\system32;C:\WINDOWS;C:\WINDOWS\System32\Wbem;C:\WINDOWS\System32\WindowsPowerShell\v1.0;C:\WINDOWS\System32\OpenSSH;C:\Program Files (x86)\NVIDIA Corporation\PhysX\Common;C:\Users\Asd\.windows-build-tools\python27;C:\Users\Asd\AppData\Local\Microsoft\WindowsApps;C:\Users\Asd\AppData\Local\Pandoc;C:\Users\Asd\AppData\Local\Programs\MiKTeX 2.9\miktex\bin\x64;C:\Program Files\wkhtmltopdf\bin;C:\Users\Asd\AppData\Local\Programs\Microsoft VS Code\bin;C:\Users\Asd\AppData\Roaming\npm;%USERPROFILE%\AppData\Local\Microsoft\WindowsApps;C:\Program Files\Git\usr\bin\vendor_perl;C:\Program Files\Git\usr\bin\core_perl
9 verbose lifecycle takenote@0.7.2~dev: CWD: C:\Test\takenote
10 silly lifecycle takenote@0.7.2~dev: Args: [ '/d /s /c', "concurrently 'npm run server' 'npm run client'" ]
11 silly lifecycle takenote@0.7.2~dev: Returned: code: 1  signal: null
12 info lifecycle takenote@0.7.2~dev: Failed to exec dev script
13 verbose stack Error: takenote@0.7.2 dev: `concurrently 'npm run server' 'npm run client'`
13 verbose stack Exit status 1
13 verbose stack     at EventEmitter.<anonymous> (C:\Users\Asd\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\index.js:332:16)
13 verbose stack     at EventEmitter.emit (events.js:315:20)
13 verbose stack     at ChildProcess.<anonymous> (C:\Users\Asd\AppData\Roaming\npm\node_modules\npm\node_modules\npm-lifecycle\lib\spawn.js:55:14)
13 verbose stack     at ChildProcess.emit (events.js:315:20)
13 verbose stack     at maybeClose (internal/child_process.js:1021:16)
13 verbose stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)
14 verbose pkgid takenote@0.7.2
15 verbose cwd C:\Test\takenote
16 verbose Windows_NT 10.0.19041
17 verbose argv "C:\\program files\\nodejs\\node.exe" "C:\\Users\\Asd\\AppData\\Roaming\\npm\\node_modules\\npm\\bin\\npm-cli.js" "run" "dev"
18 verbose node v12.18.3
19 verbose npm  v6.14.8
20 error code ELIFECYCLE
21 error errno 1
22 error takenote@0.7.2 dev: `concurrently 'npm run server' 'npm run client'`
22 error Exit status 1
23 error Failed at the takenote@0.7.2 dev script.
23 error This is probably not a problem with npm. There is likely additional logging output above.
24 verbose exit [ 1, true ]
```

--- a/package.json
+++ b/package.json
@@ -7,7 +7,7 @@
   "private": false,
   "main": "src/server/index.ts",
   "scripts": {
-    "dev": "concurrently 'npm run server' 'npm run client'",
+    "dev": "concurrently \"npm run server\" \"npm run client\"",
     "client": "cross-env NODE_ENV=development webpack serve --config config/webpack.dev.js",
     "server": "nodemon --config config/nodemon.config.json",
     "build": "cross-env NODE_ENV=production webpack --config config/webpack.prod.js",

